controller.rb

```diff
@@ -80,57 +80,49 @@ def account_choice # Asks user if they want to deposit/withdraw, view current ba
         puts @account.return_balance(@cust_id)
       end
       account_choice
-
-    elsif action == "deposit"
+    when "deposit"
       Dialog::space
       @acct_list = @bank.customer_accounts(@name, @pin) 
       Dialog::account_prompt 
-      account_info
+      Display::account_info(@acct_list)
       @acct_num = Dialog::pick_your_account
       if !@account_id_array.include?(@acct_num)
```

Comment:
When a user goes to enter their account number to either deposit or withdraw from their account, they are flamed with the following error message: 

controller.rb:89:in `account_choice': undefined method `include?' for nil:NilClass (NoMethodError)
	from controller.rb:29:in `signin'
	from controller.rb:15:in `signup_signin'
	from controller.rb:129:in `<main>'